ibis_vega_transform/vegaexpr.py

```diff
@@ -11,11 +11,61 @@
 
 import ibis
 import ibis.expr.types as it
-import pytz
 from mypy_extensions import TypedDict
 from typing_extensions import Literal
 
 from altair_transform.utils import evaljs
+import altair_transform.utils._evaljs
+
+# Monkey patch altair_transform so that boolean operators  work on ibis expression
+
+
+def not_operator(value):
+    if isinstance(value, ibis.expr.types.ValueExpr):
```

Comment:
Are there any kinds of expressions we want to cover that are more basal than `ValueExpr`? I'm trying to understand the difference between that and `ibis.Expr`.

Reply:
I had a reason for switching it, to differentiate between ValueExpr and TableExpr, but I just tried reverting it back to the normal Expr and everything seems to work. So let's go with that for now. Thanks for picking up on this!